docs/patterns/single-new-eks-observability-accelerators/single-new-eks-cost-monitoring-ingress-observability.md

```diff
@@ -0,0 +1,93 @@
+# Single Cluster Observability - Kubecost Cost Monitoring with Secure Ingress using Cognito
+
+## Architecture
+
+The following figure illustrates the architecture of the pattern we will be deploying for Single EKS cost monitoring (Kubecost) pattern with Application Load Balancer, Amazon Cognito, and a Transport Layer Security (TLS) Certificate on AWS Certificate Manager (ACM) with Amazon Route 53 hosted zone to authenticate users to Kubecost
```

Comment:
Documentation isnt looking impressive. Please write a paragraph on top on what this pattern in accomoplishing and what is the value. Secondly please many screenshots of Kubecost console showing cost monitoring across different dimensions with Kubecost.

Reply:
please see now Ela i have added another paragraph